src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1664,13 +1662,10 @@ static bool processSubscriptions
         cacheSemGive(__FUNCTION__, "update lastNotificationTime for cached subscription");
       }
     }
-
-    /* Release object created dynamically (including the value in the map created by addTriggeredSubscriptions */
-    trigs->attrL.release();
-    delete trigs;
   }
 
-  subs.clear();
+  releaseTriggeredSubscriptions(subs);
+
```

Comment:
What you see above is the fix to the 'strange' leak about triggered subscriptions.
Instead of deleting/releasing inside the loop, I call releaseTriggeredSubscriptions after the loop.
Don't ask me why the old way stopped working. I don't have a clue.
What I **do** know is that the new way works :-)